src/test/groovy/edu/ie3/simona/model/participant/BMModelTest.groovy

```diff
@@ -1,253 +0,0 @@
-  }
-
-  def "Test calculateP"() {
-    given: "date, time, a temperature and last power output and the built model"
-    // construct date and time from string
-    ZonedDateTime dateTime = ZonedDateTime.parse(time)
-
-    /* Prepare the calculation relevant data */
-    BMModel.BMCalcRelevantData relevantData = new BMModel.BMCalcRelevantData(dateTime, Sq.create(temp, Celsius$.MODULE$))
-
-    BMModel bmModel = new BMModel(
-        UUID.fromString("08a8134d-04b7-45de-a937-9a55fab4e1af"),
-        "BM Model Test",
-        OperationInterval.apply(0L, 86400L),
-        QControl.apply(new CosPhiFixed("cosPhiFixed:{(0.0,1.0)}")),
-        Sq.create(190, Kilowatts$.MODULE$),
-        bmType.getCosPhiRated(),
-        "MockNode",
-        costControlled,
-        Sq.create(bmType.opex.value.doubleValue(), EUR$.MODULE$),
-        Sq.create(0.051d, EuroPerKilowatthour$.MODULE$),
-        0.05)
-
-    // modify data store: add last output power, one hour in the past
-    bmModel._lastPower = new Some(Sq.create(lastPower, Kilowatts$.MODULE$))
-
-    when: "the power from the grid is calculated"
-    def powerCalc = bmModel.calculateActivePower(ConstantState$.MODULE$, relevantData)
-
-    then: "compare in kilowatts"
-    powerCalc - Sq.create(powerSol, Kilowatts$.MODULE$) < Sq.create(1e-12d, Kilowatts$.MODULE$)
-
-    where:
-    time                                       | temp | costControlled | lastPower || powerSol
-    '2019-01-05T05:15:00+01:00[Europe/Berlin]' | 10   | true           | -40.0d     || -49.5d           // weekend day in heating season, power increase capped by load gradient
-    '2019-01-04T05:15:00+01:00[Europe/Berlin]' | 10   | true           | -80.0d     || -70.5d           // working day in heating season, power decrease capped by load gradient
-    '2019-01-04T05:15:00+01:00[Europe/Berlin]' | -20  | true           | -182.0d    || -190d            // peek load boiler activated, max output because cost < revenues
-    '2019-01-04T05:15:00+01:00[Europe/Berlin]' | -7   | true           | -182.0d    || -190d            // close to peak load, max output because cost < revenues
-    '2019-01-04T05:15:00+01:00[Europe/Berlin]' | -7   | false          | -150.0d    || -152.16900643778735d  // close to peak load, not cost controlled but just serving heat demand
-    '2019-07-07T10:15:00+02:00[Europe/Berlin]' | 19   | true           | -10.0d     || -12.099949463243976d    // weekend day outside heating season, increase not capped
-    '2019-07-05T05:15:00+02:00[Europe/Berlin]' | 20   | true           | -20.0d     || -11.70638561892377d   // working day outside heating season, decrease not capped
-    '2019-07-06T10:15:00+02:00[Europe/Berlin]' | 20   | true           | -0.0d      || -9.5d            // weekend day outside heating season, increase capped
-    '2019-07-05T05:15:00+02:00[Europe/Berlin]' | 22   | true           | -22.0d     || -12.5d           // working day outside heating season, decrease capped
```

Comment:
Please take the comments from here and add them above the coresponding data in the Table in the scala test